### ISSUE TITLE
Features storage key is different from cut ID

### ISSUE THREAD

**Issue by MartinKocour** [lhotse-speech/lhotse#568]
Hello, 
is there any reason why `cut.features.storage_key` is different from `cut.id`?

Do you support multiple features per cut? Or do you plan to do it?

Cheers,
Martin

**Comment by pzelasko** [comment]
Yes, multiple cuts may refer to the same `Features` manifest, think of 30min recording for which you precomputed 30min worth of feature frames, and cuts refer to subsets of those.

Lhotse is ready for multi-feature-per-cut support but I haven't had the time to write a proper high-level documentation yet. [Look into unit tests here](https://github.com/lhotse-speech/lhotse/blob/master/test/cut/test_custom_attrs.py) if you want to use it, and feel free to ask questions here. The idea is that cuts can be assigned custom fields that are persistent, and you can use `FeatureWriter.store_array()` to store extra features (either temporal e.g. pitch or static e.g. ivector) and get the manifests for them. Cuts will automatically handle truncation and padding for them, and there is support for their collation in mini-batches as well (see: https://github.com/lhotse-speech/lhotse/blob/master/test/dataset/test_collation.py#L124).
